### PR TITLE
feat(page): support bookmark block in exporting html/markdown

### LINKED ISSUE #3839: support bookmark block in exporting html/markdown

### PR DESCRIPTION
**Pull request by zuoxiaodong0815**
close #3839

### CODE REVIEW

File: packages/blocks/src/__internal__/content-parser/file-exporter/exporter-style.ts [hunk 1]
```diff
@@ -121,9 +121,118 @@ export const globalCSS = css`
     --affine-palette-shape-black: #000;
     --affine-palette-shape-grey: #c2c2c2;
     --affine-tooltip: #424149;
+    --affine-shadow-1: 0px 0px 4px 0px rgba(66, 65, 73, 0.14);
```
Comment (doodlewind) [inline packages/blocks/src/__internal__/content-parser/file-exporter/exporter-style.ts:124]: This naming looks a bit off 🤔

Comment (zuoxiaodong0815) [reply packages/blocks/src/__internal__/content-parser/file-exporter/exporter-style.ts:124]: The properties and values here refer to this code, https://github.com/toeverything/design/blob/main/packages/theme/src/index.ts
image

Comment (zuoxiaodong0815) [reply packages/blocks/src/__internal__/content-parser/file-exporter/exporter-style.ts:124]: I haven't come up with a final method for css variables in exporting file.

The first way is to rewrite css variables like the current way.

The second way is to import theme package in blocksuite.

The third way is to collect all css variables when exporting file. Like following,
```typescript
  const cssAffineVars = {};
  const styles = window.getComputedStyle(document.documentElement);

  for (const prop of styles) {
    if (prop.startsWith("--affine-")) {
      cssAffineVars[prop] = styles.getPropertyValue(prop);
    }
  }
```

**Review by doodlewind — APPROVED** [review]
Let's just land this for the moment.

**Pull request merged**